R/plumber.R

```diff
@@ -818,7 +819,24 @@ plumber <- R6Class(
     #' @param req request object
     #' @details required for httpuv interface
     onHeaders = function(req) {
-      NULL
+      maxSize <- private$maxSize
+      if (maxSize <= 0)
+        return(NULL)
+
+      reqSize <- 0
+      if (length(req$CONTENT_LENGTH) > 0)
+        reqSize <- as.numeric(req$CONTENT_LENGTH)
+      else if (length(req$HTTP_TRANSFER_ENCODING) > 0)
```

Comment:
I'm not sure I see why this case is here? Appears as though this header is no longer in HTTP/2, so should we really bother here? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding

Reply:
If we do go this route, can we document the fact that it should be `option('plumber.maxRequestSize' = Inf)` to accept requests in this case?

Reply:
Currently, shiny does not even document this

https://github.com/rstudio/shiny/blob/64e09315fc5f38a31e1bf26179296a97be30b0be/R/shiny-options.R#L73-L74

Reply:
Closing this convo here. Continue in #552